api/MediaStream.json

```diff
@@ -563,9 +563,16 @@
             "edge": {
               "version_added": "12"
             },
-            "firefox": {
-              "version_added": "59"
-            },
+            "firefox": [
+              {
+                "version_added": "72"
+              },
+              {
+                "version_added": "59",
+                "partial_implementation": true,
+                "notes": "Before Firefox 72, the event handler was exposed but had no effect."
```

Comment:
How about `version_removed` instead? Otherwise it will look like this note still applies to current Firefox.